js/utils.js

```diff
@@ -983,11 +983,23 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
     });
 
     $filtersGroup.html(activeFilterElements);
-    $filtersGroup.find('.hidden-filter-controls-filter.mixitup-control-active').on('click', function(event) {
-      options.event = event;
+    $activeFiltersHolder.removeClass('hidden');
+
+    _.forEach($filtersGroup.find('.hidden-filter-controls-filter'), function(item) {
+      var $element = $(item);
+
+      $element.css({
```

Comment:
I don't understand why we are using this approach instead of using template and CSS files to achieve the result.

Reply:
@tonytlwu your right, I thought that there was no possibility of a class, since these elements with the same class exist in other places, but I was mistaken, changed